src/base-api.js

```diff
@@ -0,0 +1,75 @@
+// import axios from 'axios';
+
+// const baseURL = axios.create({
+//   baseURL: 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi',
+//   headers: {
+//     'Content-type': 'application/json, text/plain, */*',
+//   },
+// });
+
+// this next function was only to create a new app on the API
+// const newApp = () => baseURL.post('/apps/');
+
```

Comment:
- [ ] Kindly remove this commented code, it makes your code unclean, and also it does not serving any purpose in your codebase.

Reply:
> * [ ]  Kindly remove this commented code, it makes your code unclean, and also it does not serve any purpose in your codebase.

Above ☝🏼 are the comments from the previous review. They seem not to be addressed, please work on these concerns first before asking for another review. Proceeding reviews can only be performed based on what the last existing review requires, please consider this.

Reply:
The comments were deleted, but the commit it's in the history of the branch, because I misclicked the button merge, and merge it before creating the commit. Sorry! And thanks for the review!